dialects/cudf.yaml

```diff
@@ -11,3 +11,17 @@ scalar_functions:
     required_options:
       overflow: SILENT # cudf rolls over on overflow
       rounding: TIE_TO_EVEN
+  - name: multiply
+    local_name: multiply
+    required_options:
+      overflow: SILENT # cudf rolls over on overflow
+      rounding: TIE_TO_EVEN
+  - name: divide
+    local_name: divide
+    required_options:
+      overflow: SILENT # cudf rolls over on overflow
+      rounding: TIE_TO_EVEN
+      on_division_by_zero: LIMIT
+      on_domain_error: NAN
+  - name: modulus
```

Comment:
cudf also supports `mod` function on floating points, while the substrait documentation states operation on only integer arguments.

Reply:
I'll make this change in the substrait spec.